src/pages/index.js

```diff
@@ -94,37 +93,6 @@ const DescriptiveText = styled(P)`
     margin-bottom: ${size.medium};
 `;
 
-const ThumbnailCard = styled(Card)`
-    position: relative;
-`;
-
-const ThumbnailButton = styled(Button)`
-    left: 0;
-    right: 0;
-    top: 0;
-    /* Account for bottom margin and title of card */
-    bottom: 52px;
```

Comment:
This value for centering the play button was hardcoded for the use case on the index page, so we needed a more general solution for use in other places